test/routes/setup_test.js

```diff
@@ -0,0 +1,18 @@
+let path = require('path');
```

Comment:
We could also put that call to `chai.should()` inside of a `suiteSetup` block perhaps?